### ISSUE TITLE
Reviews by Category (editor): Clear all button doesn't have enough contrast

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1677]
![imatge](https://user-images.githubusercontent.com/3616980/73388703-02291180-42d3-11ea-8043-b3a20df985a8.png)

Text is `#D94F4F` and background `#F3F3F4`. It looks like contrast is not enough:

https://webaim.org/resources/contrastchecker/

**Comment by Aljullu** [comment]
Since this issue was created, the red color changed from `#d94f4f` to `#cc1818` and background from `#f3f3f4` to `#ffffff`. Now there is enough contrast. :ok_hand:
